src/daemon/device_bragi.c

```diff
@@ -233,6 +234,7 @@ int start_dongle_bragi(usbdevice* kb, int makeactive){
     // Force back to SW mode
     // FIXME: Does this make a difference?
     bragi_set_property(kb, BRAGI_MODE, BRAGI_MODE_SOFTWARE);
+    kb->active = 1;
```

Comment:
This should probably be a separate commit (before the one that introduces the pair commands)